pgcli/main.py

```diff
@@ -456,16 +470,27 @@ def _evaluate_command(self, text):
                     click.secho("Aborted!", err=True, fg='red')
                     break
 
-            if self.pgspecial.auto_expand:
-                max_width = self.cli.output.get_size().columns
+            lines, columns = self.cli.output.get_size()
+            expanded = self.pgspecial.expanded_output
+            # Checking the width of only the first content row is fast but my be faulty
+            if cur and not expanded and content_exceeds_width(cur.fetchone(), columns):
```

Comment:
This completely break `\x auto`. We can't decide based on the first row. That's why we use the return value from `tabluate` to check if it exceeds the width.